src/App.js

```diff
@@ -1,25 +1,40 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import logo from "./logo.svg";
+import "./App.css";
+import { BrowserRouter, Switch, Route } from "react-router-dom";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
+      <div className="App">
+        <header className="App-header">
+          <img src={logo} className="App-logo" alt="logo" />
+          <p>
+            Edit <code>src/App.js</code> and save to reload.
+          </p>
+          <a
+            className="App-link"
+            href="https://reactjs.org"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Learn React
+          </a>
+        </header>
+
+        <Switch>
+          <Route
+            path="/"
+            exact
+            component={"todo: insert component with list of restaurants"}
+          />
+          <Route
+            path="/restaurants/:id"
```

Comment:
I think we can drop "restaurants" part and just use `/:id`. What do you think? Since we don't have too many pages, we can go with:

`/` for listing page
`/:id` for details page

Reply:
yes, shorter is better. if we would have other pages and routes, this route should be probably in the end not to capture other requests, right?

Reply:
I made 2 commits to resolve all 3 issues. I don't see that there are added automatically to be shown here. How can I show them here just like in yours initial review with Kateryna?

Reply:
Ah, ok, there shown below